README.md

```diff
@@ -48,6 +48,7 @@ Patch locales functions to koa app.
   - {String} cookieField: locale field name on cookie. Optional, default is `locale`.
   - {String} cookieDomain: domain on cookie. Optional, default is `''`.
   - {Object} localeAlias: locale value map. Optional, default is `{}`.
+  - {Boolean} writeCookie: set cookie if header not sent. Optional, default is `true`.
```

Comment:
LGTM 👍 
just this update beacause i do the same [here](https://github.com/koajs/locales/pull/43/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R53) #43 
```suggestion
  - {Boolean} **writeCookie**: set cookie if header not sent. Optional, default is `true`.
```